project.py

```diff
@@ -28,6 +27,11 @@ def __init__(self):
         self.width = None
         self.height = None
         self.saved_cnt = None
+        self.seed = None
+        self.numberOfSlides = None
+        self.frameCounter = None
```

Comment:
@Foxiy Same as above. I could probably live with this but then you have stuff like `self.saved_cnt` in here too and that is heresy by definition.